lib/spack/spack/build_systems/python.py

```diff
@@ -399,3 +399,10 @@ def remove_files_from_view(self, view, merge_map):
                 view.remove_file(src, dst)
             else:
                 os.remove(dst)
+
+    @property
+    def site_packages_dir(self):
+        site_packages_dir = os.path.join(
+            self.prefix, self.spec['python'].package.site_packages_dir
```

Comment:
This will be the site_packages_dir for Python, not for the package being installed?

Reply:
Outdated